### ISSUE TITLE
Datalabeling tests are broken

### ISSUE THREAD

**Issue by kurtisvg** [GoogleCloudPlatform/python-docs-samples#2643]
Data labeling test are broken - likely a race condition when using a single project.
 
See https://github.com/GoogleCloudPlatform/python-docs-samples/pull/2642

**Comment by gguuss** [comment]
@kurtisvg Do you want to take this or should I? I'm happy to investigate but I saw that you'd done the change referenced in the commit but I was auto-assigned.

**Comment by kurtisvg** [comment]
Happy to let you take over if you are willing - I was only attempting to update the snippets to use the test endpoint, which is how I discovered the tests were broken.

**Comment by gguuss** [comment]
I'll take a look, currently waiting on a few other work items.

**Comment by gguuss** [comment]
Anecdotally the tests are passing under nox in Python 2.7.

**Comment by gguuss** [comment]
As mentioned in the bug this does seem to be specific to running the tests in parallel.

**Comment by gguuss** [comment]
Repro in #2662

**Comment by gguuss** [comment]
That branch got really messy, it appears to be a race condition in the teardown.

**Comment by gguuss** [comment]
Progress in #2665 I have serialized the tests in that branch to try and determine the ordering of the build-up / tear down / timing that is causing the problem. Still failing though.

**Comment by gguuss** [comment]
I was unable to resolve this, assigning to @kurtisvg

**Comment by andrewferlitsch** [comment]
@kurtisvg PTAL

**Comment by kurtisvg** [comment]
Closing in favor of buildcop issues.
